dashboard/test/ui/runner.rb

```diff
@@ -23,14 +23,16 @@
 require 'colorize'
 require 'open3'
 require 'parallel'
+require 'socket'
 
 require 'active_support/core_ext/object/blank'
 
 ENV['BUILD'] = `git rev-parse --short HEAD`
 
+GIT_BRANCH = GitUtils.current_branch
 COMMIT_HASH = RakeUtils.git_revision
 S3_LOGS_BUCKET = 'cucumber-logs'
-S3_LOGS_PREFIX = GitUtils.current_branch
+S3_LOGS_PREFIX = "#{Socket.gethostname}/#{GIT_BRANCH}"
```

Comment:
Logs on S3 from this PR's CircleCI UI tests:
![image](https://cloud.githubusercontent.com/assets/1615761/17536129/b41a7bc8-5e47-11e6-869c-39c86e8b083a.png)